packages/build/bin/run-mocha.js

```diff
@@ -31,8 +31,8 @@ function run(argv, options) {
   if (typeof options === 'boolean') options = {dryRun: options};
   options = options || {};
   if (setMochaOpts) {
```

Comment:
setMochaOpts is checking for presence of `--opts`.  So IIUC the current implementation, if `--opts` is not provided, we add `--config`. Which means that `lb-mocha --config my.json` will add `--config .mocharc.json` to the command line. Please add a test to verify this scenario.

How about backwards compatibility? If we replace `--opts` with `--config` in our check and stop detecting `--opts`, then I think this is a breaking change requiring a semver-major release. It may be more conventient for our users if we keep support for both `--opts` and `--config` options.

Reply:
Maybe it's better to bump `@loopback/build` to the next major version. Using `--opts` for mocha@6.x seems to be add confusion.

Reply:
I am fine with publishing a new semver-major version of `@loopback/build`. See https://github.com/strongloop/loopback-next/blob/master/docs/site/DEVELOPING.md#making-breaking-changes for a check-list.

Reply:
It turns out that mocha 6.x continues to support `mocha.opts`. I have improved the PR to respect `--opts` and `mocha.opts` in addition to `.mocharc.*` formats. It should be now backward compatible.